wallet/tickets.go

```diff
@@ -156,9 +166,44 @@ func (w *Wallet) VoteBitsForTicket(ticketHash *chainhash.Hash) (uint16, error) {
 
 // SetVoteBitsForTicket sets the per-ticket vote bits.  These vote bits override
 // the wallet's default vote bits.
-func (w *Wallet) SetVoteBitsForTicket(ticketHash *chainhash.Hash, voteBits uint16) error {
+func (w *Wallet) SetVoteBitsForTicket(update TicketVoteBitsUpdate) error {
```

Comment:
I'm not sure why this API changed to this instead of just changing the voteBits parameter (remember, this is the singular version, and only needs to update one ticket at a time).  I would prefer it to be:

`func (w *Wallet) SetVoteBitsForTicket(ticketHash *chainhash.Hash, voteBits stake.VoteBits) error`

I mentioned this earlier in another comment.